### PR TITLE
Getting ready for 6.5 release

### PR DESCRIPTION
**Pull request by PaulWessel**
Minor steps taken. Need help:

1. Test with GMT.jl, assumed to work fine [@joa-quim please check box if OK]
2. Test with PyGMT, assumed to work fine [@seisman, @maxrjones please check box if O]
3. Test it builds with MATLAB [@joa-quim  please check box if OK?]
4. Anything outstanding in `remote_datasets` or `gmtserver-admin `[@Esteban82 ?]
5. Test show no outstanding issues that I am not aware of.  We do not expect to reach 0 failures anyway.  OK to check those?

### CODE REVIEW

**Comment by Esteban82** [comment]
> * Anything outstanding in `remote_datasets` or `gmtserver-admin `[@Esteban82 ?]

No, I can't find anything outstanding.

**Comment by seisman** [comment]
All PyGMT tests look good except a broken test reported at https://github.com/GenericMappingTools/pygmt/issues/2511.

**Comment by yvonnefroehlich** [comment]
> > * Anything outstanding in `remote_datasets` or `gmtserver-admin `[@Esteban82 ?]
> 
> No, I can't find anything outstanding.

Hm, not sure if this is related to here, but for the remote dataset `Earth Mask`, the thumbnail image on the overview page (https://www.generic-mapping-tools.org/remote-datasets/) and the large image on the separate documentation page (https://www.generic-mapping-tools.org/remote-datasets/earth-mask.html) should be made identical (please see issue https://github.com/GenericMappingTools/remote-datasets/issues/104).

**Comment by PaulWessel** [comment]
Hi @Esteban82, would you be able to address @yvonnefroehlich points?

**Comment by Esteban82** [comment]
> Hi @Esteban82, would you be able to address @yvonnefroehlich points?

Yes, sure. What version should I use?

**Comment by PaulWessel** [comment]
None has cpts except that one so maybe none should have CPT legend

**Comment by Esteban82** [comment]
> None has cpts except that one so maybe none should have CPT legend

Well, the earth_age also have a cpt (https://www.generic-mapping-tools.org/remote-datasets/earth-age.html)

**Comment by PaulWessel** [comment]
OK, never mid. You choose!

**Comment by seisman** [comment]
@PaulWessel Changes in this PR should be merged into the master branch.

**Comment by PaulWessel** [comment]
Need an approval.

File: CITATION.cff [hunk 1]
```diff
@@ -29,9 +29,9 @@ authors:
 - family-names: "Esteban"
   given-names: "Federico"
   orcid: "https://orcid.org/0000-0002-0641-7371"
-version: 6.4.0
-doi: 10.5281/zenodo.6623271
-date-released: 2022-06-15
+version: 6.5.0
+doi: 10.5281/zenodo.10119499
+date-released: 2023-11-20
```
Comment (seisman) [inline CITATION.cff:34]: Need to update the release date.

Comment (PaulWessel) [reply CITATION.cff]: done

**Review by seisman — APPROVED** [review]

**Pull request merged**